gcloud/monitoring/client.py

```diff
@@ -195,6 +203,130 @@ def metric_descriptor(self, type_,
             display_name=display_name,
         )
 
+    @staticmethod
+    def metric(type_, labels):
+        """Construct a metric object.
+
+        The type of the Metric should match the type of a MetricDescriptor.
+        For a list of built-in Metric types and the associated labels, see:
+
+        https://cloud.google.com/monitoring/api/metrics
+
+        Here is an example of creating using a built-in metric type:
+
+             >>> type_ = 'compute.googleapis.com/instance/cpu/utilization'
+             >>> metric = client.metric(type_, labels={
+             ...    'instance_name': 'my-instance',
+             ... })
+
+        For custom metrics, the simplest way to get the type is from the
+        :class:`~gcloud.monitoring.metric.MetricDescriptor` class used to
+        create the custom metric::
+
+             >>> metric = client.metric(metric_descriptor.type, labels={
+             ...    'instance_name': 'my-instance',
+             ... })
+
+        :type type_: string
+        :param type_: The metric type name.
+
+        :type labels: dict
+        :param labels: A mapping from label names to values for all labels
+                       enumerated in the associated :class:`MetricDescriptor`.
+
+        :rtype: :class:`~gcloud.monitoring.metric.Metric`
+        :returns: The Metric created with the passed in arguments
+        """
+        return Metric(type=type_, labels=labels)
+
+    @staticmethod
+    def resource(type_, labels):
+        """Construct a resource object.
+
+        For a list of possible MonitoredResources and their associated labels,
+        see:
+
+        https://cloud.google.com/monitoring/api/resources
+
+        For example, to create a Resource object for a GCE instance::
+
+            >>> resource = client.resource('gce_instance', labels={
+            ...     'project_id': 'my-project-id',
+            ...     'instance_id': 'my-instance-id',
+            ...     'zone': 'us-central1-f'
+            ... })
+
+        :type type_: string
+        :param type_: The monitored resource type name.
+
+        :type labels: dict
+        :param labels: A mapping from label names to values for all labels
+                       enumerated in the associated
+                       :class:`ResourceDescriptor`.
+
+        :rtype: :class:`gcloud.resource.Resource`
+        :returns: The Resource created with the passed in arguments
+        """
+        return Resource(type_, labels)
+
+    @staticmethod
+    def timeseries(metric, resource, value,
```

Comment:
I think we want this to be `time_series`, for consistency with `TimeSeries`. They actually went through the trouble of switching from `Timeseries` in V2 to `TimeSeries` in V3, so let's not backslide.

It's okay that the module name is `timeseries`, and I have no problem with local variable names being spelled like that. In English, we should write "time series".